standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -3461,7 +3474,7 @@ private Table getTableInternal(String catName, String dbname, String name,
               LOG.warn("Unexpected resultset size:" + ret.size());
               throw new MetaException("Unexpected result from metadata transformer:return list size is " + ret.size());
             }
-            t = (Table)(ret.keySet().iterator().next());
+            t = (ret.keySet().iterator().next());
```

Comment:
unnecessary brackets

Reply:
yeah..someone should teach that trick to my save-action :D